xdsl/frontend/code_generation.py

```diff
@@ -71,6 +72,87 @@ def visit_Assign(self, node: ast.Assign) -> None:
         # TODO: Implement assignemnt in the next patch.
         pass
 
+    def visit_BinOp(self, node: ast.BinOp):
+        op_name: str = node.op.__class__.__name__
+
+        # Local table which maps the name of a binry operator to the
+        # corresponding Python function.
+        op_to_python = {
```

Comment:
When I first tried to understand this code, I was really confused by this. I think it'd be great if we could adjust the naming to make it more clear that this maps "Python AST Operator Name" to python function name. "Op" virtually always stands for operation in xDSL/MLIR, so using it here for operator is a bit confusion.